main.tf

```diff
@@ -119,9 +119,11 @@ module "vpc" {
   enable_dns_hostnames = true
 
   # nacl
-  manage_default_network_acl = var.manage_default_network_acl
-  default_network_acl_name   = var.default_network_acl_name
-  default_network_acl_tags   = var.default_network_acl_tags
+  manage_default_network_acl  = var.manage_default_network_acl
+  default_network_acl_name    = var.default_network_acl_name
+  default_network_acl_tags    = var.default_network_acl_tags
+  default_network_acl_ingress = var.default_network_acl_ingress
+  default_network_acl_egress  = var.default_network_acl_egress
```

Comment:
these two were missing, which caused the previous version to not be able to customise the default ACL